examples/next/getting-started/.env.local.sample

```diff
@@ -2,4 +2,4 @@
 NEXT_PUBLIC_WORDPRESS_URL=https://headlessfw.wpengine.com
 
 # Plugin secret found in WordPress Settings->Headless
-FAUSTWP_SECRET_KEY=YOUR_PLUGIN_SECRET
```

Comment:
I don't think you should change the old example here as it will fail for old faust.